autosklearn/data/feature_validator.py

```diff
@@ -13,31 +13,67 @@
 from sklearn import preprocessing
 from sklearn.base import BaseEstimator
 from sklearn.compose import make_column_transformer
+from sklearn.exceptions import NotFittedError
 
+from autosklearn.util.logging_ import PickableLoggerAdapter
 
-class FeatureValidator(BaseEstimator):
 
-    def __init__(self) -> None:
+SUPPORTED_FEAT_TYPES = typing.Union[
+    list,
```

Comment:
Shouldn't this be `typing.List`?

Reply:
I think this should be Union to indicate a logical 'or' like indication, that or a list or an array, etc is expected. A list will mean that we will expect multiple arrays as train data I think* 

please let me know if I am mistaken to fix this

Reply:
I think you had a look at the wrong line here. I meant replacing `list` in line 22 by `typing.List`.